### DISCUSSION TITLE
Hello. Will support for AMD graphics cards be added?

### DISCUSSION THREAD

**Discussion by eismorgan** [Anjok07/ultimatevocalremovergui#1062] [Q&A]
My previous NVIDIA graphics card burned out. Because of this incident I decided to upgrade and chose AMD RX570.

**Comment by saulop** [comment]
Try OpenCL version. It works great for me. Very fast indeed (and silent too :-)

**Comment by celeste-ex3** [comment]
Then switch over to The [Open Cl Version](https://github.com/Anjok07/ultimatevocalremovergui/releases/download/v5.6/UVR_v5.6.0_setup_opencl.exe) of UVR 5 the link to which is just below the main download link, as the main download link only supports Nvidia Cards, so for AMD or Intel Cards use the [Open Cl Version](https://github.com/Anjok07/ultimatevocalremovergui/releases/download/v5.6/UVR_v5.6.0_setup_opencl.exe) hope that helps.
